jujugui/static/gui/src/app/store/env/sandbox.js

```diff
@@ -319,7 +319,7 @@ YUI.add('juju-env-sandbox', function(Y) {
         Response: {
           ProviderType: state.get('providerType'),
           DefaultSeries: state.get('defaultSeries'),
-          Name: 'Sandbox'
+          Name: 'sandbox'
```

Comment:
That's odd, I'm seeing it in lowercase. Could you try `make clean-gui run` please?